processor/transform/tables-to-jsx.ts

```diff
@@ -35,7 +40,10 @@ const visitor = (table: Table, index: number, parent: Parents) => {
   };
 
   visit(table, 'tableCell', tableCellVisitor);
-  if (!hasFlowContent) return;
+  if (!hasFlowContent) {
+    table.type = 'table';
```

Comment:
Ohhh is this where the `tableau` error was coming from?

Reply:
Yea, I added a custom table component, silly-ly named `tableau`, to handle block content. I believe I ran into some weirdness the last time I tried to override a built in type.

And yea, I forgot to add a compiler or transform it or anything.